docs/pact-properties.md

```diff
@@ -392,13 +392,13 @@ table.
   ("Transfer money between accounts"
     (properties [(row-enforced 'accounts 'ks from)]))
 
-  (let ((from-bal (at 'balance (read 'accounts from)))
-        (from-ks  (at 'ks      (read 'accounts from)))
-        (to-bal   (at 'balance (read 'accounts to))))
+  (let ((from-bal (at 'balance (read accounts from)))
+        (from-ks  (at 'ks      (read accounts from)))
```

Comment:
please use `with-read` to avoid the duplicate db hit.
```(with-read accounts from { 'balance := from-bal 'ks := from-ks }```
might do it for to-bal too for consistent style, your call